.prettierrc.js

```diff
@@ -8,4 +8,5 @@ module.exports = {
   singleQuote: true,
   tabWidth: 2,
   trailingComma: 'all',
+  plugins: ['prettier-plugin-packagejson'],
```

Comment:
Not strictly required for this PR, but I noticed Prettier wasn't updating the order of scripts in `package.json`. Turns out we never added `prettier-plugin-packagejson` to the config after bumping to Prettier 3.